packages/ui/locales/es/views.json

```diff
@@ -238,6 +238,12 @@
     "replyHere": "",
     "commits": ""
   },
+  "notFound": {
+    "title": "Something went wrong…",
```

Comment:
Let’s try running this text through GPT to minimize the need to revisit it in the future.